app/components/App.js

```diff
@@ -100,6 +131,21 @@ class App extends React.Component {
 		})
 	}
 
+	updatePath(step, stepNum) {
+		const board = this.state.board;
+		const tempBoard = board.map((row) =>{
+			return row.map((cell)=>Object.assign({}, cell));
+		})
+		const [x, y] = step;
+		tempBoard[y][x].stepVisited	= stepNum;
+			
+		this.setState(() => {
```

Comment:
Arrow functions are good for brevity:
```js
this.setState(() => ({ board: tempBoard, }));
```
Better: name your temp board 'board' and use ES2015 shorthand object literal
```js
this.setState(() => ({ board, }));
```
Even better, specifically for React's setState, which can accept an object literal to merge with current state:
```js
this.setState({ board, });
```